lib/internal/map.js

```diff
@@ -1,8 +1,7 @@
 import noop from 'lodash/noop';
-import once from './once';
 
 export default function _asyncMap(eachfn, arr, iteratee, callback) {
-    callback = once(callback || noop);
+    callback = callback || noop;
```

Comment:
nope, `each`/`eachLimit`/`eachSeries` gaurentee that we'll only ever be calling callback once